Sources/Common/Background Queue/Queue.swift

```diff
@@ -123,11 +123,17 @@ public class CioQueue: Queue {
     }
 
     public func addTrackInAppDeliveryTask(deliveryId: String, event: InAppMetric) -> ModifyQueueResult {
-        addTask(type: QueueTaskType.trackDeliveryMetric.rawValue,
-                data: TrackDeliveryEventRequestBody(type: .inApp,
-                                                    payload: DeliveryPayload(deliveryId: deliveryId,
-                                                                             event: event,
-                                                                             timestamp: dateUtil.now)))
+        addTask(
+            type: QueueTaskType.trackDeliveryMetric.rawValue,
+            data: TrackDeliveryEventRequestBody(
+                type: .inApp,
+                payload: DeliveryPayload(
+                    deliveryId: deliveryId,
+                    event: event,
+                    timestamp: dateUtil.now
+                )
+            )
+        )
```

Comment:
Was this style change an automatic linting deal or is this an update to personal/squad code-style? It doesn't look relevant to this push (other than making things nicer as you're here), so I figured I would ask.

Reply:
I did make a commit in this PR for formatting swift code. We use git hooks and a tool (swiftformat) that formats code the same for everyone on the team, automatically. 

However, if a developer on our team doesn't have git hooks installed, code formatting does not automatically happen for them. Which introduces unformatted code to the codebase. This is what happened here which is why I introduced a commit to run the formatter.